app/src/main/res/layout/activity_bill.xml

```diff
@@ -255,32 +268,69 @@
                             android:textSize="@dimen/sp_10"
                             android:textStyle="bold"/>
 
-                        <ImageView
+                        <android.support.design.widget.TextInputLayout
                             android:layout_width="wrap_content"
                             android:layout_height="wrap_content"
-                            android:layout_marginTop="@dimen/dp_2"
-                            android:onClick="@{()-> viewModel.onAddBillClick()}"
-                            app:srcCompat="@drawable/ic_cart_60dp"
-                            tools:ignore="ContentDescription"/>
+                            bind:errorText="@{viewModel.formError}">
+
+                            <android.support.v7.widget.CardView
+                                android:layout_width="wrap_content"
+                                android:layout_height="wrap_content"
+                                android:paddingRight="@dimen/dp_10"
```

Comment:
WARNING: [RtlHardcoded] Consider adding `android:paddingEnd="@dimen/dp_10"` to better support right-to-left layouts
 Using `Gravity#LEFT` and `Gravity#RIGHT` can lead to problems when a layout is rendered in locales where text flows from right to left. Use `Gravity#START` and `Gravity#END` instead. Similarly, in XML `gravity` and `layout_gravity` attributes, use `start` rather than `left`.

For XML attributes such as paddingLeft and `layout_marginLeft`, use `paddingStart` and `layout_marginStart`. *NOTE*: If your `minSdkVersion` is less than 17, you should add *both* the older left/right attributes *as well as* the new start/right attributes. On older platforms, where RTL is not supported and the start/right attributes are unknown and therefore ignored, you need the older left/right attributes. There is a separate lint check which catches that type of error.

(Note: For `Gravity#LEFT` and `Gravity#START`, you can use these constants even when targeting older platforms, because the `start` bitmask is a superset of the `left` bitmask. Therefore, you can use `gravity="start"` rather than `gravity="left|start"`.)